cfgov/v1/blocks.py

```diff
@@ -276,6 +277,44 @@ class NavGroup(blocks.StructBlock):
         label='Menu items')
 
 
+class NavFooter(blocks.StructBlock):
+    draft = blocks.BooleanBlock(
+        required=False,
+        default=False,
+        help_text='If checked, this block will only show '
+        'on our sharing site (Content).',
+        label='Mark block as draft'
+    )
+    content = blocks.RichTextBlock(
+        required=False,
+        features=['link']
+    )
+
+    def clean(self, value):
+        cleaned = super(NavFooter, self).clean(value)
+        if cleaned['content']:
+            html = expand_db_html(cleaned['content'].source)
+            cleaned['content'] = RichText(
+                self.accessible_links(html)
+            )
+        return cleaned
+
+    @staticmethod
+    def accessible_links(html):
```

Comment:
Do you think you could add a docstring here to describe specifically what this function alters?

Reply:
[Docstring](https://github.com/cfpb/cfgov-refresh/blob/458bc21149d7015f1cc94264e099c0d24452044a/cfgov/v1/models/menu_item.py#L110-L115) added in https://github.com/cfpb/cfgov-refresh/pull/4267/commits/3c4d5a22a78f54ecdc0dc864930f6d3f68c6869f